content/home/featured-carousel.yaml

```
@@ -1,25 +1,138 @@
-      icon: "FaUser"
+
+  - title: "Mesoamerican Migration Project"
+    categories: ["Sociology", "Web Development", "Migration"]
+    description: "This project serves as a public-facing website to see/download data files from MMP-led research projects. The main goal of this project is to move all assets from Princeton to Brown and to create a more responsive website that the PI can later edit.\n\nTwo phases of the project:\n\n- Facelift of the old site\n- Data storage and PI edit"
+    image: "/images/home/featured-carousel/mmp.png"
+    organizations:
+      - name: "Department of Sociology"
+        organization: "Population Studies and Training Center"
+        pi: ["David Lindstrom"]
+        icon: "FaUser"
+      - name: "Center for Demographic, Urban and Environmental Studies"
+        organization: "El Colegio de México"
+        pi: ["Silvia Giorguli Saucedo", "Jéssica Natalie Nájera Aguirre"]
+        icon: "FaUser"
     buttons:
       - text: "Website"
-        url: "https://provident-study.org"
-        variant: "primary_filled" 
\ No newline at end of file
+        url: "https://mmp.research.brown.edu/mmp"
+        variant: "primary_filled"
+
+  - title: "Collider Parts Database"
+    categories: ["Web Development", "Particle Physics"]
+    description: |
+      An app that tracks the construction of particle detector modules across 4 different research institutions. The database records include information about each change, when it happened, and who made the change. Users can search the database, build modules, and upload images taken during production. The application uses a reusable component-based JavaScript frontend and deployed with a serverless cloud-based backend.
+    image: "/images/home/featured-carousel/collider.png"
+    organizations:
+      - name: "Department of Physics"
+        organization: "Brown University"
+        pi: ["Ulrich Heintz"]
+        icon: "FaUser"
+    buttons:
+      - text: "Website"
+        url: "https://collider-parts-db.web.app/"
+        variant: "primary_filled"
+
+  - title: "Gearshift Fellowship"
+    categories: ["Web Development", "jsPsych", "Cognitive Science"]
+    description: |
+      The Gearshift Fellowship is a car driving game where players take the role of a driver. The purpose of the game is to complete different missions and collect as many reward points as possible in the setting of 5 missions, each with different goals/challenges.
+
+      The game is developed using HTML, CSS, and Javascript. The experiment running the game and facilitating data recording is built on jsPsych, a JavaScript framework used in web-based behavioral experiments.
+
+      Honeycomb, an open-source repository that integrates cognitive science and web development practices to create deployable psychophysiological tasks compatible with lab equipment recordings.
+    image: "/images/home/featured-carousel/gearshift.gif"
+    organizations:
+      - name: "TODO"
```

This is the department? Is that good enough for org?

```suggestion
      - name: "Cognitive and Psychological Sciences"
```